packages/esm-patient-labs-app/src/lab-orders/add-lab-order/lab-order-form.scss

```
@@ -1,10 +1,9 @@
-@use '@carbon/styles/scss/type';
-@use '@carbon/styles/scss/spacing';
+@use '@carbon/type';
+@use '@carbon/layout';
 @use '@carbon/colors';
 @import '@openmrs/esm-styleguide/src/vars';
 @import "../../root";
```

While we're cleaning this up...

```suggestion
@use '@openmrs/esm-styleguide/src/vars' as *;
```